aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -174,48 +144,35 @@ const data = [
 
 const StepThree = () => {
   return (
-    <>
-      <Box gap="small" margin={{ bottom: 'large' }}>
-        <List data={data} pad={{ horizontal: 'none', vertical: 'small' }}>
-          {(datum, index) => (
-            <Box key={index} direction="row" gap="medium" align="center">
-              <Checkmark color="text-strong" size="small" />
-              <Text color="text-strong">{datum}</Text>
-            </Box>
-          )}
-        </List>
-        <Text color="text-strong">
-          Include guidance to what will occur when “Finish Setup” is clicked.
-        </Text>
-      </Box>
-      <Button fill="horizontal" label="Finish Setup" primary type="submit" />
-    </>
+    <Box gap="small" margin={{ bottom: 'medium' }}>
+      <List data={data} pad={{ horizontal: 'none', vertical: 'small' }}>
+        {(datum, index) => (
+          <Box key={index} direction="row" gap="medium" align="center">
+            <Checkmark color="text-strong" size="small" />
+            <Text color="text-strong">{datum}</Text>
+          </Box>
+        )}
+      </List>
+      <Text color="text-strong">
```

Comment:
I am just noticing this, but I think we need the Wizard Figma file to be cleaned up. It looks like it is manually overriding the font colors and weights, rather than using the componentized fonts. This will come back and bite us when folks come to reference and reconcile.

I believe this font should be our regular text color. The checkmark list may be appropriate as text-strong + medium (500) weight.

Reply:
That is a good note regarding the color overrides. I will touch base with Vicky or Greg on that today.

Reply:
I've opened an issue to track the design discussions: #1302